### PR TITLE
[MTL-2018] - registration now open, add french text

### PR DESCRIPTION
**Pull request by pdion891**
Registration now open.
add latest sponsor
update french text.

To website maintainers: how could we translate the "SPONSOR" title in the welcome page  so we have a french and english term for it ?  

Thanks!

### CODE REVIEW

**Comment by MikeRosTX** [comment]
@pdion891 give us a :+1: if the navbar French spelling is good for you.
**[https://deploy-preview-4624--devopsdays-web.netlify.com/events/2018-montreal/welcome/](https://deploy-preview-4624--devopsdays-web.netlify.com/events/2018-montreal/welcome/)**

**Comment by bridgetkromhout** [comment]
The build is taking a while, but I put in corrections that will allow the links to work.

**Comment by bridgetkromhout** [comment]
Hi, @pdion891 - hopefully this is what you need? Please let us know if we can help make it what you're looking for. I'm also going to ask @phrawzty to take a look to make sure this is as good as it can be.

To be clear, we're fine with merging this when @pdion891 agrees.

File: content/events/2018-montreal/welcome.md [hunk 1]
```diff
@@ -79,7 +90,7 @@ Description = "devopsdays Montreal 2018"
     <strong>Contact</strong>
   </div>
   <div class = "col-md-8">
-    {{< event_link page="contact" text="Get in touch with the organizers" >}}
+    {{< event_link page="contact" text="Get in touch with the organizers / Contacter les organizateurs" >}}
```
Comment (phrawzty) [inline content/events/2018-montreal/welcome.md]: organisateurs (avec un _s_), non ? :)

File: data/events/2018-montreal.yml [hunk 2]
```diff
@@ -26,19 +26,29 @@ registration_link: "https://www.eventbrite.ca/e/devops-day-montreal-2018-tickets
...
+  - name: Location - Localisation
+    url: "/events/2018-montreal/location/"
+    icon: "map-o"
+  - name: Registration - Inscription
+    url: "/events/2018-montreal/registration/"
+    icon: "user-plus"
+  - name: Propose - Proposer
+    url: "https://www.papercall.io/devopsdays-mtl-2018"
+    icon: "lightbulb-o"
+  - name: Sponsor - Sponsoriser
```
Comment (phrawzty) [inline data/events/2018-montreal.yml]: Vous utilisez « commanditer » ailleurs - pourquoi pas ici ?

**Comment by phrawzty** [comment]
There are a bunch of strings left to be translated ([for example](https://github.com/devopsdays/devopsdays-web/pull/4624/files#diff-a5cefce4497f926880c52638a03f6b28R45)). Imho this should be merged as-is in order to get the info live, and the local team can iterate over the rest of the strings if they so choose.

**Pull request merged**